unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/GltfContainer/GltfContainerHandler.cs

```diff
@@ -166,6 +166,16 @@ private static List<Collider> SetUpPhysicColliders(MeshFilter[] meshFilters)
             return physicColliders;
         }
 
+        // Compatibility layer for old GLTF importer and GLTFast
+        private static bool IsCollider(MeshFilter meshFilter)
+        {
+            string ownName = meshFilter.name.ToLower();
```

Comment:
I know it's not followed here but you can use `StringComparison.CurrentCultureIgnoreCase` at least in your case to avoid extra allocation from `ToLower`: just pass it as a second argument to `Contains`